src/contract-details/ContractDetailsCard.js

```diff
@@ -61,9 +60,11 @@ export default class ContractDetailsCard extends Component {
 						/>)
 					</button> :
 					<div>
-					{validationError.message ?
-						validationError.message.split(')')[1] :
-						validationError }
+					{(!!validationError) &&
```

Comment:
You don't need the () and also you don't need to convert it to boolean with the !!.
In fact it is better not to.